doc/user-manual.md

```diff
@@ -1,18 +1,264 @@
-  - [JVM Memory](#JVM-Memory)
-  - [JVM GC](#JVM-GC)
-  - [Kafka Client](#Kafka-Client)
-  - [RabbitMq Producer](#RabbitMq-Producer)
-  - [RabbitMq Consumer](#RabbitMq-Consumer)
+  
+# User Manual
 
-## Metric Detail
+- [User Manual](#user-manual)
+  - [Agent.properties](#agentproperties)
+    - [Internal HTTP Server](#internal-http-server)
+    - [Metric](#metric)
+    - [Kafka](#kafka)
+    - [HTTP Reqeust Metric](#http-reqeust-metric)
+    - [JDBC SQL Metric](#jdbc-sql-metric)
+    - [JDBC Connection Metric](#jdbc-connection-metric)
+    - [RabbitMQ Metric](#rabbitmq-metric)
+    - [Kafka Metric](#kafka-metric)
+    - [Redis Metric](#redis-metric)
+    - [JVM GC Metric](#jvm-gc-metric)
+    - [JVM Memory Metric](#jvm-memory-metric)
+    - [SQL MD5Dictionary](#sql-md5dictionary)
+    - [Tracing](#tracing)
+    - [Logging](#logging)
+  - [Prometheus Support](#prometheus-support)
+  - [Health Check and Readiness Check Endpoint](#health-check-and-readiness-check-endpoint)
+  - [Tracing](#tracing-1)
+    - [Tracing Component](#tracing-component)
+    - [Custom Span Tag](#custom-span-tag)
+      - [JDBC](#jdbc)
+      - [Cache](#cache)
+      - [RabbitMQ Producer And Consumer](#rabbitmq-producer-and-consumer)
+      - [Kafka Producer And Consumer](#kafka-producer-and-consumer)
+  - [Metric](#metric-1)
+    - [Metric Key](#metric-key)
+      - [Type And SubType Detail](#type-and-subtype-detail)
+    - [Metric Field](#metric-field)
+      - [HTTP Request](#http-request)
+      - [JDBC Statement](#jdbc-statement)
+      - [JDBC Connection](#jdbc-connection)
+      - [JVM Memory](#jvm-memory)
+      - [JVM GC](#jvm-gc)
+      - [Kafka Client](#kafka-client)
+      - [RabbitMq Producer](#rabbitmq-producer)
+      - [RabbitMq Consumer](#rabbitmq-consumer)
+  
+## Agent.properties
+Extracting the default configuration file
+```
+$ jar xf easeagent.jar agent.properties log4j2.xml
+```
+Run the user application with EaseAgent
+```
+$ export EASE_AGENT_PATH=[Replace with agent path]
+$ java "-javaagent:${EASE_AGENT_PATH}/easeagent.jar=${EASE_AGENT_PATH}/agent.properties" -jar user-app.jar
+```
+
+### Internal HTTP Server
+EaseAgent opens port `9900` by default to receive configuration change notifications and Prometheus requests.
+
+Key| Default Value | Description |
+---| ---| ---|
+`easeagent.server.enabled` | true | Enable Internal HTTP Server. `False` can disable it. EaseAgent will no longer accept any HTTP requests (`Prometheus`、`Health Check`、`Readiness Check`) when the Internal HTTP Server is disabled. User can add VM parameter:`-Deaseagent.server.enabled=[true or false]` to override.|
+`easeagent.server.port` | 9900 | Internal HTTP Server port. User can add VM parameter:`-Deaseagent.server.port=[new port]` to override. |
+
+### Metric
+Key| Default Value | Description |
+---| ---| ---|
+`observability.metrics.enabled` | true | Enable all metrics collection. `False`: Disable all metrics collection |
+
+### Kafka
```

Comment:
Why we need to config Kafka address, need a shorten text to answer this question.